inputs/templates/terra_workspaces/cohort_mode/samples_1kgp_156.tsv.tmpl

```diff
@@ -0,0 +1,157 @@
+entity:sample_id	bam_or_cram_file
+NA10847	gs://fc-56ac46ea-efc4-4683-b6d5-6d95bed41c5e/CCDG_13607/Project_CCDG_13607_B01_GRM_WGS.cram.2019-02-06/Sample_NA10847/analysis/NA10847.final.cram
```

Comment:
Sure. This one matches the order of the 312-sample one - I could sort that too, although then it would mix the reference panel samples with the rest of the samples, and wouldn't match the batch assignment order